nxdrive/data/qml/AccountsTab.qml

```diff
@@ -6,233 +6,94 @@ import "icon-font/Icon.js" as MdiFont
-        ScaledText { text: qsTr("USERNAME") + tl.tr; color: label }
         GridLayout {
+            id: accountCreation
             columns: 2
+            columnSpacing: 15
 
             anchors {
-                left: parent.left
-                leftMargin: 185
+                right: parent.right
+                top: parent.top
+                topMargin: 15
+                rightMargin: 25
             }
 
-            // Account icon
-            IconLabel {
-                icon: MdiFont.Icon.accountCircle
-                iconColorDisabled: primaryBg
-                enabled: false
-            }
-
-            // Dropdown list
-            AccountsComboBox {
-                id: accountSelect
-                Layout.maximumWidth: 400
+            NuxeoButton {
+                // Add a new account
+                Layout.alignment: Qt.AlignRight
+                text: qsTr("NEW_ENGINE") + tl.tr
+                onClicked: newAccountPopup.open()
             }
         }
-    }
-
-    StackLayout {
-        visible: hasAccounts
-        currentIndex: accountSelect.currentIndex
 
+    // The EngineModel list
+    Rectangle {
+        Layout.fillWidth: true;
+        Layout.fillHeight: true
+        height: parent.height
+        width: parent.width
         anchors {
-            top: accountSelection.bottom
+            top: accountCreation.bottom
             left: parent.left
-            topMargin: 35
+            topMargin: 20
             leftMargin: 30
+            bottomMargin: 20
         }
-
-        Repeater {
-            id: accounts
-            model: EngineModel
-            delegate: GridLayout {
-                columns: 2
-                columnSpacing: 50
-                rowSpacing: 20
-                property bool authenticated: !api.has_invalid_credentials(uid)
-                property bool sync_enabled: feat_synchronization.enabled
-                property string forceUi: force_ui || wui
-
-                // Server URL
-                ScaledText { text: qsTr("URL") + tl.tr; color: label }
-                Link {
-                    text: server_url
-                    onClicked: api.open_remote_server(accountSelect.getRole("uid"))
-                }
-
-                // Server UI (Web-UI or JSF)
-                ScaledText {
-                    text: qsTr("SERVER_UI") + tl.tr
-                    color: label
-                    Layout.alignment: Qt.AlignTop
-                }
+        Flickable {
+            id: enginesList
+            anchors.fill: parent
+            clip: true
+            contentHeight: engines.height + accountCreation.height + 50
+            width: parent.width
+            ScrollBar.vertical: ScrollBar {}
+            flickableDirection: Flickable.VerticalFlick
+            boundsBehavior: Flickable.StopAtBounds
+            ListView {
+                id: engines
+                width: parent.width;
+                height: contentHeight
+                spacing: 20
```

Comment:
Fix for macOS. It do not know why it works for GNU/Linux and Windows, but anyway:
```suggestion
                spacing: 20
                interactive: false
```
Apply the same fix to `SynchronizationTab.qml` please.